pkgs/tools/filesystems/bcachefs-tools/default.nix

```diff
@@ -6,13 +6,13 @@ assert fuseSupport -> fuse3 != null;
 
 stdenv.mkDerivation {
   pname = "bcachefs-tools";
-  version = "2020-11-17";
+  version = "2021-05-20";
```

Comment:
```suggestion
  version = "unstable-2021-05-20";
```

Reply:
I'm not sure how useful a signal this is considering no stable version exists?

Reply:
Taking a closer look at the [manual](https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming) it seems like it belongs into the name instead.